prime-router/metadata/schemas/WATERS/waters-covid-19.schema

```diff
@@ -0,0 +1,217 @@
+    csvFields: [{ name: patientSex, format: $display}]
+
+  - name: ordering_provider_zip_code
+    csvFields: [{ name: orderingProviderZip}]
+
+  - name: testing_lab_zip_code
+    csvFields: [{ name: performingFacilityZip}]
+
+  - name: ordering_provider_street
+    csvFields: [{ name: orderingProviderAddress}]
+
+  - name: ordering_provider_street2
+    csvFields: [{ name: orderingProviderAddress2}]
+
+  - name: ordering_provider_city
+    csvFields: [{ name: orderingProviderCity}]
+
+  - name: ordering_provider_state
+    csvFields: [{ name: orderingProviderState}]
+
+  - name: ordering_provider_phone_number
+    csvFields: [{ name: orderingProviderPhone}]
+
+  - name: first_test
+    csvFields: [{ name: firstTest}]
+
+  - name: previous_test_type
+    type: TEXT
+    csvFields: [{ name: previousTestType}]
+
+  - name: previous_test_date
+    type: DATE
+    csvFields: [{ name: previousTestDate}]
+
+  - name: previous_test_result
+    type: TEXT
+    csvFields: [{ name: previousTestResult}]
+
+  - name: employed_in_healthcare
+    csvFields: [{ name: healthcareEmployee}]
+
+    # Edited.  Was 'healthcareEmployeeType - as I thought patient_role was more general.
+    # Eg, Staff, Resident, Visitor, Student are the values used in SimpleReport.
+  - name: patient_role 
+    csvFields: [{name: Patient_role}]
+    type: TEXT
+
+  - name: symptomatic_for_disease
+    csvFields: [{ name: symptomatic}]
+
+  - name: symptoms_list   # Edited.  Was 'patient_symptom_onset'
+    type: TEXT
+    csvFields: [{ name: symptomsList}]
+
+  - name: hospitalized
+    csvFields: [{ name: hospitalized}]
+
+  - name: hospitalized_test_result    # Edited.  Was 'hospitalized_code'.  The example value was a 'test_result' valueset value.
+    type: CODE
+    valueSet: covid-19/test_result
+    documentation: The result of the test performed. For IgG, IgM and CT results that give a numeric value put that here.
+    csvFields: [{ name: hospitalizedCode}]
+
+  - name: icu
+    csvFields: [{ name: symptomsIcu}]
+
+  - name: resident_congregate_setting
+    csvFields: [{ name: congregateResident}]
+
+  - name: site_of_care     # Edited name and now using the site_of_care valueset.
+    type: CODE
+    valueSet: site_of_care
+    csvFields: [{ name: congregateResidentType}]
+
+  - name: pregnant    # This appeared twice.  I eliminated one.  Edited.  was 'patient_pregnant'
+    csvFields: [{ name: pregnant}]
+
+  - name: patient_email
+    csvFields: [{ name: patientEmail}]
+
+  - name: ordered_test_name
+    csvFields: [{ name: testName}]
+
+  - name: abnormal_flag
+    csvFields: [{ name: testResultText}]
+
+  - name: test_performed_code
+    csvFields: [{ name: testPerformed}]
+
+  - name: ordering_provider_last_name
+    csvFields: [{ name: orderingProviderLname}]
+
+  - name: ordering_provider_first_name
+    csvFields: [{ name: orderingProviderFname}]
+
+  - name: patient_phone_area_code   # Edited this.
+    type: TEXT
+    csvFields: [{ name: patientPhoneAreaCode}]
+
```

Comment:
Area code is not a separate element from the phone number. Pretty sure that this will result in RS sending messages without area codes.

Reply:
I put a note in the schema to work this out with Waters team.